cpu/cc26x2_cc13x2/include/cc26x2_cc13x2_rf_params.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2020 Locha Inc
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup         cpu_cc26x2_cc13x2
+ * @{
+ *
+ * @file
+ * @brief           CC26x2/CC13x2 IEEE 802.15.4 netdev driver
+ *
+ * @author          Jean Pierre Dudey <jeandudey@hotmail.com>
+ */
+
+#ifndef CC26X2_CC13X2_RF_PARAMS_H
+#define CC26X2_CC13X2_RF_PARAMS_H
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define CC26X2_CC13X2_RF_PA_ENTRY(bias, gain, boost, coefficient) \
+    ((bias) << 0) | ((gain) << 6) | ((boost) << 8) | ((coefficient) << 9)
+
+/**
+ * @brief   RF PA table entry
+ */
+typedef struct {
+    uint8_t dbm;  /**< PA entry value in dBm */
```

Comment:
Element `dbm` should be signed type, dbm is set to negative value in `cc26x2_cc13x2_rf_patable` array. Leading to value overflow when trying to set dbm power lower than 0.

Reply:
Solved, thanks for pointing it up :-)